rabin-karp.cpp

```diff
@@ -0,0 +1,67 @@
+#include<bits/stdc++.h>
```

Comment:
We don't use any inbuilt libraries in here. Kindly read [contributing.md](https://github.com/moghya/dsalglib/blob/master/contributing.md). Use already implemented functions as per requirement.